package.json

```diff
@@ -39,6 +40,7 @@
     "react-redux": "^8.0.2",
     "swr": "^1.3.0",
     "usehooks-ts": "^2.6.0",
+    "yup": "^0.32.11",
```

Comment:
Why use `yup`, we have `zod`, I think we should use one package

Reply:
ok will need to see how we can integrate `zod` with `react-hook-form`

Reply:
got it. it's similar implementation just like yup

Reply:
done